### DISCUSSION TITLE
Dúvida sobre a função open no Capítulo 9

### DISCUSSION THREAD

**Discussion by chicortiz** [clube-de-leitura-python/pense-em-python-2021#32] [Perguntas & Respostas] [answered]
Olá pessoal tudo bem?

Estou tentando executar o primeiro comando sugerido no capítulo 9. Estou tentando inclusive com a lista aberta em cima como "words.py"

O Shell está retornando o seguinte:

Python 3.6.9
>>>  fin = open('words.txt')
  File "<pyshell>", line 1
    fin = open('words.txt')
    ^
SyntaxError: unexpected indent

>>>  fin = open('words.txt')
  File "<pyshell>", line 1
    fin = open('words.txt')
    ^
SyntaxError: unexpected indent

>>>  fin = open('words.py')
  File "<pyshell>", line 1
    fin = open('words.py')
    ^
SyntaxError: unexpected indent

>>>

**Comment by omyasuda** [comment, marked as answer]
'''

>>> fin = open('words.txt')

>>> type(fin)
<class '_io.TextIOWrapper'>
>>> 


'''

Para mim, parece funcionar bem....

dá uma verificada na versão do Thonny e do python em uso....

![Thoy-about-2021-12-31 12-21-00](https://user-images.githubusercontent.com/1220078/147830602-5a919bd8-f1b1-46e1-9095-fd4f928713d7.png)

será que esta sua instalação de thonny tem algum problema?!?!

**Reply by chicortiz** [reply]
Olá Yasuda! Muitíssimo obrigado! No livro há um espaço antes do "fin" que gera o erro. Eliminando este espaço obti o seguinte:

Python 3.6.9
>>> fin = open('words.txt')
>>> fin.readline()
'aa\n'
>>> 

:)

**Reply by omyasuda** [reply]
opa.... legal que você conseguiu se virar....
este problema com os espaços em branco adicionais é mesmo muito chato.
